From 3310151ea5bab0ce0296ccf938991f38e8b602e1 Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Sun, 17 Feb 2019 02:28:57 -0600
Subject: [PATCH 2/2] SUW: Don't make google suw use material_light

The material_light theme causes gms to crash with "Error inflating
class TextView" when attempting to do a device to device restore in
google suw. This change allows it to use the default glif_light.

Change-Id: I287f201799aba81c19869000e3aadb47ef67299b
---
 res/values/strings.xml | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/res/values/strings.xml b/res/values/strings.xml
index a1954a2..a00793c 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
      Copyright (C) 2013-2015 The CyanogenMod Project
-     Copyright (C) 2017 The LineageOS Project
+     Copyright (C) 2017,2019 The LineageOS Project
 
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
@@ -22,8 +22,6 @@
     <string name="lineage_wizard_script_uri" translatable="false">android.resource://org.lineageos.setupwizard/raw/lineage_wizard_script</string>
     <string name="lineage_wizard_script_user_uri" translatable="false">android.resource://org.lineageos.setupwizard/raw/lineage_wizard_script_user</string>
 
-    <!-- Partner SUW Strings -->
-    <string name="theme_type" translatable="false">material_light</string>
     <string name="activity_label_empty" />
 
     <string name="next">Next</string>
-- 
2.17.1

